### PR TITLE
Multistage build with release

### PR DESCRIPTION
**Pull request by adityapatwardhan**

### CODE REVIEW

File: tools/releaseBuild/yaml/compliance.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    iterationPath: 'One\_Default'
+    uploadFortifySCA: false
+    uploadFxCop: false
+    uploadModernCop: false
+    uploadPREfast: false
+    uploadRoslyn: false
+    uploadTSLint: false
+
+- task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
+  displayName: 'Component Detection'
```
Comment (TravisEz13) [inline tools/releaseBuild/yaml/compliance.yml]: posix compliance

Comment (adityapatwardhan) [reply tools/releaseBuild/yaml/compliance.yml]: What is posix compliance?

Comment (TravisEz13) [reply tools/releaseBuild/yaml/compliance.yml]: new lines at the end of the files

Comment (adityapatwardhan) [reply tools/releaseBuild/yaml/compliance.yml]: Fixed.

**Pull request merged**